### PR TITLE
PRS 6625 2612452

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Revision of entries, one deleted

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]
I approve but check if the mention of LOC3703Gwegwe present in the earlier record is relevant

**Comment by DenisNosnitsin1970** [comment]
I am adding it.

File: new/PRS13452Mamas.xml [hunk 1]
```diff
@@ -53,6 +53,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                     <persName xml:lang="gez" xml:id="n1">ማማስ፡</persName>
                       <persName xml:lang="gez" type="normalized" corresp="#n1">Māmās</persName>
+                      <persName xml:lang="gez" xml:id="n2" type="alt">ማማስ፡</persName>
```
Comment (eu-genia) [inline new/PRS13452Mamas.xml:56]: I would not restate the same name as "alt", there is no difference in spelling - I would put it all together

```
<persName xml:lang="gez" xml:id="n1">ማማስ፡</persName>
                      <persName xml:lang="gez" type="normalized" corresp="#n1">Māmās</persName>
<persName xml:lang="gez" corresp="#n1" type="alt">Māmmās</persName>
```

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13452Mamas.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/PRS13452Mamas.xml:56]: Ah,ok.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**